packages/@aws-cdk/aws-batch/README.md

```diff
@@ -242,10 +242,14 @@ new batch.JobDefinition(stack, 'batch-job-def-from-local', {
 
 ### Importing an existing Job Definition
 
-To import an existing batch job definition, call `JobDefinition.fromJobDefinitionArn()`.
+To import an existing batch job definition, call `JobDefinition.fromJobDefinitionArn()` or `JobDefinition.fromJobDefinitionName()`.
 
 Below is an example:
 
 ```ts
 const job = batch.JobDefinition.fromJobDefinitionArn(this, 'imported-job-definition', 'arn:aws:batch:us-east-1:555555555555:job-definition/my-job-definition');
+
+// OR
+
+const job = batch.JobDefinition.fromJobDefinitionName(this, 'imported-job-definition', 'my-job-definition');
```

Comment:
Maybe add a note saying that using the job definition name without a revision will import the *latest active* revision?

This is according to the service API [docs](https://docs.aws.amazon.com/batch/latest/APIReference/API_SubmitJob.html#Batch-SubmitJob-request-jobDefinition):

> The job definition used by this job. This value can be one of name, name:revision, or the Amazon Resource Name (ARN) for the job definition. If name is specified without a revision then the latest active revision is used.

And also show an example of importing with a revision. 

I know its not strictly related to your PR, but seems like a good chance to add it.

Reply:
Yeah that sounds good. Let me add.